zebra-chain/src/transaction/serialize.rs

```diff
@@ -433,7 +440,7 @@ impl ZcashSerialize for Transaction {
                 outputs.zcash_serialize(&mut writer)?;
                 lock_time.zcash_serialize(&mut writer)?;
                 writer.write_u32::<LittleEndian>(expiry_height.0)?;
-                writer.write_i64::<LittleEndian>(*value_balance)?;
+                writer.write_i64::<LittleEndian>(i64::from(*value_balance))?;
```

Comment:
is there a reason that `value_balance.into()` doesn't work here? (either way is fine, it doesn't matter, i'm just curious since i've run into issues with Into and references)

Reply:
it could be into, just needed to put parens around the deref